### PR TITLE
SF3 support

### PR DESCRIPTION
**Pull request by oussaka**

### CODE REVIEW

**Comment by mremi** [comment]
Hi @oussaka ,

Thanks for your contribution, can you fix the CI please?

**Comment by oussaka** [comment]
CI has been fixed.

Tes remarques/critiques sur la PR sont les bienvenues !

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 composer.lock
 phpunit.xml
 vendor
+.idea
```
Comment (mremi) [inline .gitignore]: Should be in your own global .gitignore file, best practice because not part of this package

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
 
-before_script: composer install --dev --prefer-source
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.5
+    - php: 5.6
+    - php: 7.0
+    - php: 7.1
+    - php: hhvm
```
Comment (mremi) [inline .travis.yml]: I think we can remove support of hhvm and add the one for 7.2

File: .travis.yml [hunk 3]
```diff
@@ -1,10 +1,25 @@
...
+
+before_install:
+  - if [[ "$TRAVIS_PHP_VERSION" != "hhvm" ]]; then phpenv config-rm xdebug.ini; fi
+  - composer self-update
+
+install:
+  - composer install
+
+script:
+  - ./vendor/bin/phpunit
```
Comment (mremi) [inline .travis.yml]: Missing blank line at the end, best practice

File: composer.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
     "name": "mremi/contact-bundle",
     "type": "symfony-bundle",
     "description": "Implementation of a contact form for Symfony2",
-    "keywords": ["Symfony2", "contact", "form"],
+    "keywords": ["Symfony3", "contact", "form"],
```
Comment (mremi) [inline composer.json]: Just `Symfony` is better IMO

**Review by mremi — COMMENTED** [review]
Thx @oussaka , can you also please squash your commits to have only one?

File: .gitignore [hunk 5]
```diff
@@ -1,4 +1,4 @@
 .php_cs.cache
 composer.lock
 phpunit.xml
-vendor
+vendor
```
Comment (mremi) [inline .gitignore]: Can you re-add the final blank line please?

File: Resources/config/controller.xml [hunk 6]
```diff
@@ -5,7 +5,7 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="mremi_contact.contact_controller" class="Mremi\ContactBundle\Controller\ContactController">
+        <service id="mremi_contact.contact_controller" class="Mremi\ContactBundle\Controller\ContactController" public="true">
```
Comment (mremi) [inline Resources/config/controller.xml]: Why make it public?

File: Resources/config/routing.xml [hunk 7]
```diff
@@ -1,15 +1,13 @@
...
         <default key="_controller">mremi_contact.contact_controller:indexAction</default>
     </route>
 
-    <route id="mremi_contact_confirmation" pattern="/contact/confirmation">
+    <route id="mremi_contact_confirmation" path="/contact/confirmation" methods="GET">
         <default key="_controller">mremi_contact.contact_controller:confirmAction</default>
-        <requirement key="_method">GET</requirement>
     </route>
-</routes>
+</routes>
```
Comment (mremi) [inline Resources/config/routing.xml]: Final blank line please

**Comment by mremi** [comment]
Can you add new entry in `CHANGELOG.md` please?

**Comment by mremi** [comment]
Thx @oussaka 👍

**Pull request merged**